packs/bit-bucket/requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.6.0
```

Comment:
Ok, I will use that and then generate a new PR. Going for the bit-bucket API library.